### PR TITLE
[MUL-280] Add validation for category name

### PR DESCRIPTION
**Pull request by Quiet0o**
- Add validation rule to prevent add empty message category

### CODE REVIEW

**Review by DominikWolos — COMMENTED** [review]
LGTM

File: src/Component/Messaging/Resources/validation/Category.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<constraint-mapping xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (senghe) [inline src/Component/Messaging/Resources/validation/Category.xml:2]: No OS header exception :)

File: src/Component/Messaging/Resources/validation/Category.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+                    xmlns="http://symfony.com/schema/dic/constraint-mapping"
+                    xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping
+                    http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
+    <class name="BitBag\OpenMarketplace\Component\Messaging\Entity\Category">
+        <property name="name">
+            <constraint name="NotBlank"/>
```
Comment (senghe) [inline src/Component/Messaging/Resources/validation/Category.xml:8]: Additionally, please add some behats here :)

**Review by senghe — CHANGES_REQUESTED** [review]

File: features/admin/message_categories.feature [hunk 3]
```diff
@@ -0,0 +1,18 @@
+@message_category
```
Comment (senghe) [inline features/admin/message_categories.feature:1]: `@messaging` :pray:

File: tests/Behat/Resources/suites/ui/admin/message_categories.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    suites:
+        ui_message_category:
+            contexts:
+                - open_marketplace.behat.context.ui.admin.product_listing
+                - Behat\MinkExtension\Context\MinkContext
+                - sylius.behat.context.setup.shop_security
+                - open_marketplace.behat.context.setup.vendor
+                - sylius.behat.context.hook.doctrine_orm
+            filters:
+                tags: '@message_category&&@ui'
```
Comment (senghe) [inline tests/Behat/Resources/suites/ui/admin/message_categories.yml:11]: `@messaging` :pray:

**Review by senghe — CHANGES_REQUESTED** [review]

File: features/admin/message_categories.feature [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  As an Administrator
+  I need to be able to validate message category
+
+  Background:
+    Given there is an admin user "admin" with password "admin"
+    And I am logged in as an admin
+
+  @ui
+  Scenario: Incorrect message category name
+    And I am on "/admin"
```
Comment (senghe) [inline features/admin/message_categories.feature:13]: "When" section starts here

File: tests/Behat/Resources/suites.yml [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
-    - suites/ui/admin/disabling_vendors.yml
-    - suites/ui/admin/editing_vendors.yml
+    - suites/vendor/customer_dashboard.yml
+    - suites/vendor/vendor_register.yml
+    - suites/vendor/vendor_update.yml
+    - suites/vendor/vendor_commission.yml
+    - suites/ui/vendor/product_listing.yml
+    - suites/ui/admin/product_listing.yml
+    - suites/ui/admin/message_categories.yml
+    - suites/start_conversation.yml
```
Comment (senghe) [inline tests/Behat/Resources/suites.yml:9]: Why this changes?

**Review by senghe — CHANGES_REQUESTED** [review]

File: tests/Behat/Resources/suites.yml (hunk 6 above)
Comment (Quiet0o) [reply tests/Behat/Resources/suites.yml]: Already fixed it was totally unnecessary change

**Review by senghe — APPROVED** [review]

**Pull request merged**